routes/index.js

```diff
@@ -337,10 +350,18 @@ module.exports = function (passport) {
       let adminEmail = req.user.email;
       let adminBadge = req.user.badge;
       dbAPI.logEvent('administration', adminBadge, `Admin ${adminName} (${adminEmail}) has deleted user with badge ID: ${deletedBadge}`);
-    });
+      dbAPI.logEvent('user traffic', deletedBadge, `User with badge ID: ${deletedBadge} has been deleted by Admin ${adminName}`);
+      req.flash('success', 'The user has been successfully deleted from the system');
+      req.flash('fade_out', '3000');
```

Comment:
Since the flash is inside a promise and promises return immediately and continue execution is there a case where it call `res.redirect` below before it has time to flash the message? I am not sure myself. Perhaps we could move that `res.redirect` to be after the flash. Similar to the catch below.